python/lsst/sims/photUtils/Photometry.py

```diff
@@ -15,458 +15,110 @@
 from collections import OrderedDict
 import lsst.utils
 from lsst.sims.photUtils import Sed, Bandpass, LSSTdefaults, calcGamma, \
-                                calcMagError_m5, PhotometricParameters, magErrorFromSNR
+                                calcMagError_m5, PhotometricParameters, \
+                                magErrorFromSNR, BandpassDict
 from lsst.sims.utils import defaultSpecMap
 
-__all__ = ["PhotometryHardware", "PhotometryBase"]
+__all__ = ["loadBandpassesFromFiles", "loadTotalBandpassesFromFiles"]
```

Comment:
Should have a separate module for these two functions? It would seem that we should add these two as as classmethods to BandPass and BandPassDict. Also, does it make sense to put both of those in a single file?